### DISCUSSION TITLE
:octocat: Collaboration Julien/LiamM32 topic

### DISCUSSION THREAD

**Discussion by julien-boudry** [julien-boudry/Condorcet#156] [Collaboration]
@LiamM32

Just give you access to Experimental/* branches. Maybe you can develop your Schulze STV here, could be simpler.

**Comment by LiamM32** [comment]
Thank you. I really hope I don't accidentally push anything to one of the other branches.

**Comment by julien-boudry** [comment]
@LiamM32 #157 I'm here this evening

**Reply by LiamM32** [reply]
Sorry. I missed you. But I am now aware of your Discord server.
